Gopkg.toml

```diff
@@ -0,0 +1,34 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
+
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
```

Comment:
Hey @akashshinde . I've used this YAML library before because it works smoothly with both, JSON and YAML: https://github.com/ghodss/yaml. Also, IIRC the OpenShift project uses it as well. I cannot recall the details for why I chose it over the gopkg.in/yaml.v2 project, but please try to find out yourself which one is better and come back with a report.